<launch>
  <include file="$(find robot_trajectory_controller)/launch/robot_trajectory_controller_jaka.launch">
  </include>

  <include file="$(find jaka_driver)/launch/robot_star.launch">
</include>

  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" > 
     <arg name="port" value="8080"/>
  </include>

  <!--
  <node pkg="aliencontrol" type="aliencontrol" name="aliencontrol_top">
    <param name="cmd" value="/home/wr-1604/wr-3d-camera-main/vnc.sh"/>
  </node>
  -->
  
  <!--
  <node pkg="moveit_ros_visualization" type="moveit_joy.py" output="screen" name="moveit_joy"/>
  -->

  <node pkg="tf" type="static_transform_publisher" name="pick1" args="0.035 0.075 0.02 1.57 0 0 target pick1 100" />
  <node pkg="tf" type="static_transform_publisher" name="pick2" args="0.035 0.075 0.02 -1.57 0 0 target pick2 100" />
  <node pkg="tf" type="static_transform_publisher" name="pick3" args="0.035 0.075 -0.02 1.57 3.14 0 target pick3 100" />
  <node pkg="tf" type="static_transform_publisher" name="pick4" args="0.035 0.075 -0.02 -1.57 3.14 0 target pick4 100" />

  <node pkg="tf" type="static_transform_publisher" name="prepick1" args="0 0 0.05 0 0 0 pick1 prepick1 100" />
  <node pkg="tf" type="static_transform_publisher" name="prepick2" args="0 0 0.05 0 0 0 pick2 prepick2 100" />
  <node pkg="tf" type="static_transform_publisher" name="prepick3" args="0 0 0.05 0 0 0 pick3 prepick3 100" />
  <node pkg="tf" type="static_transform_publisher" name="prepick4" args="0 0 0.05 0 0 0 pick4 prepick4 100" />

</launch>
